src/main/java/de/tum/in/www1/artemis/domain/lecture/AttachmentUnit.java

```diff
@@ -45,4 +45,15 @@ public Attachment getAttachment() {
     public void setAttachment(Attachment attachment) {
         this.attachment = attachment;
     }
+
+    /**
+     * Removes information from a lecture unit that is not needed in the course dashboard
+     */
+    @Override
+    public AttachmentUnit slimDownForDashboard() {
```

Comment:
I find using slim down as verb here a bit odd, trim or prune etc sounds better.

Reply:
I found it funny :( 
Changed in ca06e1c5c52d5e45eb184fa2e405b36ef7533aea

Reply:
Sorry :(